resources/js/Pages/Admin/Integrations/GitHub/components/AddRepositoryModal.tsx

```
@@ -0,0 +1,246 @@
+    if (!selectedRepository || !selectedProviderId) return;
+
+    // Submit using Inertia
+    post(route('admin.integrations.github.add-repository'), {
+      onSuccess: () => {
+        onClose();
+      },
+      onError: (errors) => {
+        console.error('Error adding repository:', errors);
+
+        // Check if we got already exists message
+        if (errors.message && errors.message.includes('already exists')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)